README.md

```diff
@@ -144,29 +144,78 @@ To apply the __buttontabs__ styling, add a `o-tabs--buttontabs` class to the roo
 <ul data-o-component="o-tabs" class="o-tabs o-tabs--buttontabs" role="tablist">
 ```
 
-or use the mixin:
+The __buttontabs__ style comes in two sizes that conform to the `o-buttons` sizes: medium and big. Medium is the default size and big can be applied by adding the class `o-tabs--big`.
+
+#### Options
+
+* __Align right__: Add `o-tabs--alignright` to the root element.
+
+#### Silent mode
+
+As with all Origami components, o-tabs has a [silent mode](https://origami.ft.com/spec/v1/sass/#sass-silent-mode). To use its compiled CSS (rather than incorporating its mixins into your own Sass) set `$o-tabs-is-silent: false;` in your Sass before you import the o-tabs Sass:
```

Comment:
We've been deleting silent mode documentation from READMEs in favour of the mono mixin. E.g.
https://github.com/Financial-Times/o-table#sass
https://github.com/Financial-Times/o-forms#sass